lib/pghero/methods/data_protection.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module PgHero
+  module Methods
+    module DataProtection
+      def anonymize_query(query)
```

Comment:
Let's call this `normalize_query` and move it to the `Basic` module.

Reply:
It may conflict with `def normalize_query(query)` from QueryStats, that's why I gave it another name. 
I'll move it to `Basic`.

Reply:
Ah, missed the name conflict. Sounds good.